src/pages/index.tsx

```diff
@@ -110,6 +123,87 @@ const Home: NextPage = () => {
           </Box>
         </Box>
       </BgImage>
+      <Box
```

Comment:
This looks like a lot of content. Can we move summarize it in another component?